### PR TITLE
Feature: Genesis Validator Set

### PR DESCRIPTION
**Pull request by talekhinezh**

### CODE REVIEW

File: radix-engine/src/engine/node_properties.rs [hunk 1]
```diff
@@ -17,8 +17,13 @@ impl VisibilityProperties {
...
         node_id: RENodeId,
     ) -> bool {
-        if !mode.eq(&ExecutionMode::Application) {
-            return false;
+        match mode {
+            ExecutionMode::LoggerModule => match node_id {
+                RENodeId::Logger => return true,
+                _ => return false,
+            },
+            ExecutionMode::Application => {}
```
Comment (iamyulong) [inline radix-engine/src/engine/node_properties.rs:25]: This fall-through is not read-friendly. Possible to combine with the following match?

**Review by iamyulong — APPROVED** [review]
LGTM with minor comments.

File: radix-engine/src/transaction/transaction_receipt.rs [hunk 2]
```diff
@@ -42,6 +41,8 @@ pub struct CommitResult {
     pub state_updates: StateDiff,
     pub entity_changes: EntityChanges,
     pub resource_changes: Vec<ResourceChange>,
+    pub application_logs: Vec<(Level, String)>,
```
[the remark this answers is not in the document]
Comment (talekhinezh) [reply radix-engine/src/transaction/transaction_receipt.rs:44]: Yup agreed. I think we'll need to figure out the right abstraction between logging and events as logs for debugging shouldn't necessarily be part of persisted event stream.

File: radix-engine/src/engine/node_properties.rs (hunk 1 above)
Comment (talekhinezh) [reply radix-engine/src/engine/node_properties.rs:25]: hrm, it's not a fall-through as this match exits the match block whereas the following match returns immediately. Unless you mean something else?

Comment (talekhinezh) [reply radix-engine/src/engine/node_properties.rs:25]: ah i got you. Yup sure will update this.

**Pull request merged**